user/open-source-repository-migration.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Beta - Migrating your open source repositories to travis-ci.com
+layout: en
+---
+
+> Hi there! If you'd like to be part of this beta testing process, send us an email to support@travis-ci.com with the subject "Open Source Beta Testing Migration" - we'll be glad to send you further details!
+
+## Details and questions
+
+### Which information will be transferred to travis-ci.com?
+
+When a repository is migrated, the following information will be transferred to travis-ci.com:
+* Environment variables (public and encrypted)
+* Repository SSH keys (default or custom)
+* Repository settings
+* Last build status - the build badge in travis-ci.com will reflect this status until new builds are triggered
+* Next build number
+
+At this early stage of the beta testing process, it isn't yet possible to migrate Cron jobs. We are working to make this available as soon as possible.
+
+#### Which information will not be transferred to travis-ci.com?
+
+The following information will not be transferred to travis-ci.com when a repository is migrated.
+
+* Cron jobs configured for the repository
+* Build history - it will be migrated at a later time
+* Caches - they will be re-created automatically on the first build on travis-ci.com
+
+### Are there any manual changes required?
+
+* If you're including **build badges** in your repository, you will need to update the URL to reflect the new repository setup in travis-ci.com.
+
+* If you're using the **"Protected Branches" feature in GitHub** to require your Travis CI to pass before merging a Pull Request, you will need to update these settings to require the new GitHub Checks events instead of the old Commit Statuses that travis-ci.org was sending to your project. See: [Required status checks at GitHub](https://help.github.com/articles/about-required-status-checks/).
+
+* If you use the **Travis CLI or the Travis CI API**, you will need to make sure your requests are addressed to the correct endpoint (`api.travis-ci.com` instead of `api.travis-ci.org`).
+
+### What will happen to my travis-ci.org repository?
+
+Your repository in travis-ci.org will be automatically deactivated (this is, it will no longer receive GitHub events) and will remain available in **read-only mode**.
+
+With this read-only mode:
+* All the previous build history will be accessible at travis-ci.org
+* The "Trigger build" functionality won't be available in travis-ci.org
+* It won't be possible to restart a previous job that ran in travis-ci.org
+* The Settings page for your repository will become inactive - the transferred repository settings will now be available in travis-ci.com instead.
+
+In the future, when all the previous build history is migrated, we will provide redirections from all API/web requests going to travis-ci.org to travis-ci.com.
+
+### How long does the migration process take?
+
+The migration process should not take more than a couple seconds.
```

Comment:
Is this per repository? Or per account? :) If possible, I'd like to manage expectations here :)